### ISSUE TITLE
"upload JSON character sheet" not working

### ISSUE THREAD

**Issue by dllahr** [Mottokrosh/Sheet#78]
When I try to use this page it doesn't work.  I opened the Chrome dev console to investigate, and saw this error message when the page loads:
```javascript
angular.js:10160 Error: [ng:areq] Argument 'UploadCtrl' is not a function, got undefined
http://errors.angularjs.org/1.2.32/ng/areq?p0=UploadCtrl&p1=not%20a%20function%2C%20got%20undefined
    at http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:78:12
    at assertArg (http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:1511:11)
    at assertArgFn (http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:1521:3)
    at http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:7341:9
    at link (http://charactersheet.co.uk/pathfinder/scripts/aecaa445.modules.js:1:12104)
    at nodeLinkFn (http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:6779:13)
    at compositeLinkFn (http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:6156:13)
    at publicLinkFn (http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:6052:30)
    at boundTranscludeFn (http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:6176:21)
    at controllersBoundTransclude (http://charactersheet.co.uk/pathfinder/bower_components/angular/angular.js:6800:18) <div class="container ng-scope" ng-view="">
```
Selecting a file and clicking the upload button does nothing else visible on the page or in the dev console.

I'm using Chrome Version 88.0.4324.150 (Official Build) (x86_64), on Mac OS X High Sierra (10.13.6 (17G14042))

**Comment by Undoinglimb** [comment]
I have same issue

**Comment by edreeseg** [comment]
This should be addressed with #98 , which is ready for review. I don't believe there are other considerations beyond simply embedding the necessary script, as Angular isn't my forte, but I'm happy to dig a little deeper if necessary.

**Comment by maxluoXIII** [comment]
Fixed by #98
